docs/guidelines/logging-conventions.md

```diff
@@ -35,11 +35,22 @@ Ensure that log messages are appropriate in content and severity:
 * Endpoint (switch ID and port) must be logged using underscore as a separator, e.g.: ``` 00:00:00:00:00:00:00:01_2 ```
 * Always override toString() to give a good representation of the object.
 
+### Logging of system errors and exceptions
+* Logging of exceptions with severity WARN, ERROR or higher must be supplied with the exception stacktrace,
```

Comment:
This requirement will kill logging as an idea. Most times when you need to log some error, you know the exact reason for this error and this reason is mentioned in the log message, the stack trace will be completely excess.

Mentioned example, shows incorrect/incomplete error handling - you should log file name error message and error code returned by OS, not traceback.